### PR TITLE
feat(notifier): links subscription to plan

### PR DESCRIPTION
**Pull request by jurajpiar**
Adds related plan to subscription.
Reduces number of requests.

### CODE REVIEW

**Review by 0xartem — APPROVED** [review]
lgtm

File: src/services/notifier/models/subscription.model.ts [hunk 1]
```diff
@@ -62,4 +64,10 @@ export default class SubscriptionModel extends Model {
 
   @BelongsTo(() => ProviderModel)
   provider!: ProviderModel
+
+  // @HasOne(() => PlanModel, 'id')
+  // plan!: PlanModel
```
Comment (itofarina) [inline src/services/notifier/models/subscription.model.ts]: Is this comment needed or forgot to remove?

**Review by itofarina — APPROVED** [review]
LGTM

File: src/services/notifier/models/subscription.model.ts (hunk 1 above)
Comment (jurajpiar) [reply src/services/notifier/models/subscription.model.ts]: 🦅👁‍🗨 the latter

**Pull request merged**